src/neo/Plugins/IConsensusProvider.cs

```diff
@@ -0,0 +1,9 @@
+using Neo.Wallets;
+
+namespace Neo.Plugins
+{
+    public interface IConsensusProvider
+    {
+        void Start(Wallet wallet);
```

Comment:
Add `Stop` and `IsWatchOnly`?

Reply:
Why a consensus node need to stop?

Reply:
`IsWatchOnly` can be easily checked if you have the wallet.

Reply:
Sounds good, Erik